src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const form = document.querySelector('.form');
+
+form.addEventListener('submit', handleEvent);
+
+function handleEvent(event) {
+  event.preventDefault();
+  form.reset();
+}
```

Comment:
I don't think that this is a good idea to clear the form on submit, don't use this handler, leave the common form submit flow